### PR TITLE
File Added : Uploading asignment7 and 8

### PR DESCRIPTION
**Pull request by arshad05786**

### CODE REVIEW

**Review by roshnidurugwar — APPROVED** [review]

**Review by Oshin-250197 — APPROVED** [review]

**Review by yogesh141293 — APPROVED** [review]

File: arshad_khan/html/asignment7/home.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <style>
+            div {
+                height: 800px;
+                width: 100%;
+                /* border: 2px solid green;
```
Comment (shivling123) [inline arshad_khan/html/asignment7/home.html]: remove this comment

File: arshad_khan/html/asignment7/newuserlogin.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <style>
+            div {
+                height: 600px;
+                width: 100%;
+                /*background-color: aqua;
```
Comment (shivling123) [inline arshad_khan/html/asignment7/newuserlogin.html]: remove comment msg

File: arshad_khan/html/asignment7/newuserlogin.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                        <label><input type="text" /></label>
+                        <br /><br />
+                        <label for="lname">Last Name:</label>
+                        <label><input type="text" /></label>
+                        <br /><br />
+                        <label for="Password">Password:</label>
+                        <label><input type="password" /></label>
+                        <br /><br />
+                        <label for="gender">Gender:</label>
+                        <label><input type="radio" name="gender" value="male" /> Male</label>
```
Comment (shivling123) [inline arshad_khan/html/asignment7/newuserlogin.html]: put here option box 
remove input field

File: arshad_khan/html/asignment7/newuserlogin.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+                        <label><input type="radio" name="gender" value="female" /> Female</label>
+                        <label><input type="radio" name="gender" value="female" /> Other</label>
+                        <br /><br />
+                        <label for="email">Email id:</label>
+                        <label for=""><input type="email" /></label><br /><br />
```
Comment (shivling123) [inline arshad_khan/html/asignment7/newuserlogin.html]: remove this input no need here
all line before and after input tag no need label tag
for all places

File: arshad_khan/html/asignment7/professionaldetails.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                    </nav>
+                </center>
+            </header>
+            <h2>This is Professionaldetails Page...</h2>
+            <center>
+                <fieldset>
+                    <legend>PROFESSIONALS DETAILS</legend>
+                    <form action="login">
+                        <label for="fname">First Name:</label>
+                        <label><input type="text" /></label>
```
Comment (shivling123) [inline arshad_khan/html/asignment7/professionaldetails.html]: remove label tag
 <label><input type="text" /></label>

before and after input tag make this changes in all assesmment

**Review by shivling123 — CHANGES_REQUESTED** [review]
remove label tag
 <label><input type="text" /></label>

before and after input tag make this changes in all assesmment
same mistake in all assesment

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**